extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/menu/AuthProcessor.java

```diff
@@ -0,0 +1,274 @@
+package io.quarkus.devui.deployment.menu;
```

Comment:
This should probably live in the security extension and not in vertx. Config lives in vertx as it's core to quarkus (there is not extension, everybody has it) but not everyone will add security.

Reply:
This must live in vertx-http extension because that's where HTTP permissions are.

Reply:
However it can be in different package if you deem it more suitable.

Reply:
We don't want the menu item there if there will be no effect if you configure it, that is why I thought it should live in security somewhere.